app/assets/javascripts/admin/addon/components/admin-config-area-card-section.gjs

```diff
@@ -0,0 +1,50 @@
+import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import { action } from "@ember/object";
+import DButton from "discourse/components/d-button";
+import icon from "discourse/helpers/d-icon";
+import { i18n } from "discourse-i18n";
+
+export default class AdminConfigAreaCardSection extends Component {
+  @tracked collapsed = this.args.collapsed;
+
+  get computedHeading() {
+    if (this.args.heading) {
+      return i18n(this.args.heading);
+    }
+    return this.args.translatedHeading;
+  }
```

Comment:
I think we're moving away from this pattern of allowing a translation key OR the translation itself to just always expecting the translation itself. It's really simple for consumers of this component to do `@heading={{i18n "some.key"}}`, which is only 9 characters longer than `@heading="some.key"`, however supporting the latter form adds a bit of unnecessary complexity to the component.